src/components/AddNode.tsx

```diff
@@ -78,17 +79,20 @@ type EditorProps = {
 };
 
 function Editor({ onCreateNode, onClose }: EditorProps): JSX.Element {
+  const { setIsInputElementInFocus } = useInputElementFocus();
   const ref = React.createRef<ReactQuill>();
 
   useEffect(() => {
     (ref.current as ReactQuill).focus();
+    setIsInputElementInFocus(true);
```

Comment:
still unresolved

Reply:
its only necessary, because when clicking on another addNote element, isBlur() of the old element sometimes is called after focus() of the new element - its a corner case

Reply:
double-checked it. i removed it - it seems like it doesn't matter when isBlur is called